docs/en/pid-eaa-issuance.rst

```diff
@@ -664,17 +661,18 @@ A DPoP-bound Access Token is provided by the PID/(Q)EAA Token endpoint as a resu
     - [`DPoP-draft16 <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-16>`_. Section 6.1] and [:rfc:`7638`].
 
 
-
 Credential endpoint
 -------------------
 
-The Credential Endpoint issues a Credential as approved by the End-User upon presentation of a valid Access Token representing this approval, as defined in `OPENID4VCI`_.
+The Credential Endpoint issues a credential as approved by the End-User upon presentation of a valid Access Token representing this approval, as defined in `OPENID4VCI`_.
```

Comment:
@peppelinux , should credential be a defined term?

Reply:
no, since it is defined in this section with a dedicated section